Resources/Locale/en-US/nyanotrasen/psionics/psychic-feedback.ftl

```diff
@@ -0,0 +1,20 @@
+# Feedback messages for Focused Metapsionic Pulse
```

Comment:
Hmm. How intended is this to be meta-able? Is this intended to be vague? Is this intended to be easily learnable and guessable?

Reply:
It's intended to be vague and metaphorical, without explicit 1:1 answers, but players who play Mantis a few times will learn them fairly easily.